### DISCUSSION TITLE
Building MindsDB on my computer as a beginner OSS contributor

### DISCUSSION THREAD

**Discussion by marsidmali** [mindsdb/mindsdb#5064] [Get help]
Hello everyone, I am a beginner with limited knowledge on OSS contributing, and I'm interested in contributing to MindsDB. Can someone please guide me on how to build MindsDB on my computer so that I can start working on the code?
I've found these steps here:                                                                                                               
https://docs.mindsdb.com/contribute?utm_medium=community&utm_source=github&utm_campaign=mindsdb%20repo                                                                                                                                                                  
After you find the issue that you want to contribute to, follow the fork-and-pull workflow:

1.Fork the MindsDB repository
2.Clone the repository locally
3.Make changes and commit them
4.Push your local branch to your fork
5.Submit a Pull Request so that we can review your changes6.
6.Write a commit message
7.Make sure that the CI tests are GREEN

I've forked and cloned the repo locally but I don't know what to do next to build and run the project. I noticed there's a setup.py file. Can someone please guide me on what commands to use in the terminal to build and run MindsDB in a Python IDE?
Any help would be appreciated. Thank you!

**Comment by mariosajdini** [comment]
Here you can find a guide on how to install mindsdb https://docs.mindsdb.com/contribute/install.Keep in mind that you should have 3.7 or 3.8 python version
